src/ActivationFunctionManager.cpp

```diff
@@ -1,24 +1,30 @@
 #include "ActivationFunctionManager.hpp"
 
-#include "IdentityActivationFunction.hpp"
-#include "LogisticActivationFunction.hpp"
-#include "GaussianActivationFunction.hpp"
 
 
 
-ActivationFunctionInterface * ActivationFunctionManager::provideActivationFunction(const std::string idcode){
-        
-    if (idcode == (new IdentityActivationFunction)->getIdCode()){
-        return new IdentityActivationFunction;
-    }
-    
-    if (idcode == (new LogisticActivationFunction)->getIdCode()){
-        return new LogisticActivationFunction;
-    }
-    
-    if (idcode == (new GaussianActivationFunction)->getIdCode()){
-        return new GaussianActivationFunction;
+namespace std_actf{
+
+
+    IdentityActivationFunction id_actf = IdentityActivationFunction();
+    LogisticActivationFunction lgs_actf = LogisticActivationFunction();
+    GaussianActivationFunction gss_actf = GaussianActivationFunction();
+
+
+    ActivationFunctionInterface * provideActivationFunction(const std::string idcode){
+        if (idcode == id_actf.getIdCode()){
```

Comment:
I think this might better be done via dictionary and switch case. But overall I'm not 100% sure about this actf manager and maybe open a card, so for this PR let's leave it like that.